src/zenml/zen_server/zen_server_api.py

```diff
@@ -265,6 +316,20 @@ async def dashboard(request: Request) -> Any:
     return templates.TemplateResponse("index.html", {"request": request})
 
 
+@app.get("/system/last-activity")
+async def get_last_activity() -> Dict[str, str]:
+    """Get the timestamp of the last user activity.
+
+    Returns:
+        A JSON with last user activity.
+    """
+    return {
+        "last_user_activity": zen_store()
+        .get_server_settings()
+        .last_user_activity.isoformat()
+    }
+
+
```

Comment:
I will only make one suggestion here: instead of creating yet another endpoint, you could just add the `last_user_activity` field to the server info, which is already a non-authorized endpoint.

Reply:
@stefannica wouldnt that report back to the telemetry needlessly? not sure it matters

Reply:
@stefannica @htahir1 shall I do the change?

Reply:
I think the suggestion is very good actually. I will apply now.

Reply:
ok then lets go for it @avishniakov